—

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
What is this file for??

Reply:
Yes, I deleted the file src/IMAG because it was not necessary, and the file PGP  I think it's mongo file

Reply:
Hmm not sure, could you check if it still works after deleting it?
@miguelriverasarmiento

Reply:
I removed it and then it did not work and I put it back and it does not work either